app/Http/Controllers/UserController.php

```diff
@@ -29,7 +28,12 @@ class UserController extends Controller
     protected $positionRepository;
     protected $workspaceRepository;
 
-    public function __construct(UserRepository $userRepository, ProgramRepository $programRepository, PositionRepository $positionRepository, WorkspaceRepository $workspaceRepository)
+    public function __construct(
+        UserRepository $userRepository,
+        ProgramRepository $programRepository,
+        PositionRepository $positionRepository,
+        WorkspaceRepository $workspaceRepository
+    )
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line